### PR TITLE
Nits for updateStateMixins

### PR DESCRIPTION
**Pull request by darkwing**
Another small nit to knock down  code.

### CODE REVIEW

File: src/core/a-entity.js [hunk 1]
```diff
@@ -121,12 +121,11 @@ var proto = Object.create(ANode.prototype, {
...
       // Remove the mixins that are gone on update
       diff.forEach(function (mixinId) {
-        var forEach = Array.prototype.forEach;
-        // State Mixins
-        var stateMixinsEls = document.querySelectorAll('[id^=' + mixinId + '-]');
-        var stateMixinIds = [];
-        forEach.call(stateMixinsEls, function (el) { stateMixinIds.push(el.id); });
-        stateMixinIds.forEach(self.unregisterMixin.bind(self));
+        // State mixins
+        var stateMixinsEls = Array.from(document.querySelectorAll('[id^=' + mixinId + '-]'));
```
Comment (dmarcos) [inline src/core/a-entity.js]: It seems that the support of `Array.from` is more limited than the previous form.

**Comment by darkwing** [comment]
If we want to support old browsers without `Array.from`, we should back this out (https://github.com/aframevr/aframe/pull/1642) and use `Array.prototype.slice` (https://davidwalsh.name/nodelist-array).

**Comment by dmarcos** [comment]
Yes. I think we should get rid of Array.from. We can open a different PR to remove the one introduced by #1642

**Comment by darkwing** [comment]
Updated.  Feel free to just close if you don't want it

**Comment by darkwing** [comment]
Closing in favor of another.

**Pull request closed without merging**